src/main/java/com/github/packageurl/PackageURLBuilder.java

```diff
@@ -48,19 +50,31 @@ private PackageURLBuilder() {
      * Obtains a reference to a new builder object.
      *
      * @return a new builder object
+     * @since 1.1.0
      */
     public static PackageURLBuilder aPackageURL() {
         return new PackageURLBuilder();
     }
 
+    private static PackageURLBuilder toBuilder(PackageURL packageURL) {
+        return PackageURLBuilder.aPackageURL()
+            .withType(packageURL.getType())
+            .withNamespace(packageURL.getNamespace())
+            .withName(packageURL.getName())
+            .withVersion(packageURL.getVersion())
+            .withQualifiers(packageURL.getQualifiers())
+            .withSubpath(packageURL.getSubpath());
+    }
```

Comment:
Nice! :100:

This way current code does not use the deprecated one.

Reply:
You can make this method package-private and use it from the deprecated method.

There should be no warning if you use implement a deprecated method in terms of a non deprecated one, the warning should only appear in the inverse situation.

Reply:
I fixed it by calling the `aPackageURL`. But, I think I need to make the `percent` methods package-private for testing?

Reply:
Possibly yes, although testing package-private method might be an indication that an abstraction is missing (some sort of `public` but internal PercentCodec).